modules/store/src/connextStore.ts

```diff
@@ -60,8 +63,24 @@ export class ConnextStore implements IClientStore {
           ),
         );
         break;
+      }
 
-      case StoreTypes.File:
+      case StoreTypes.Database: {
```

Comment:
I feel like Sequelize is a better name for this. Because we can have other database storage wrappers

Reply:
But it doesn't actually require Sequelize, it can be instantiated with just a Postgres connection URL.

Reply:
yeah but that connection url is then instantiated with sequelize so technically it can be any Sequelize compatible database but let's say I want to use mongodb or faunadb, they are not Sequelize compatible